Licensing_Program/userfiles_handling.py

```diff
@@ -7,11 +7,10 @@
 import re
 import tempfile
 
-import license_handling
-
-
-HEADER_SIGNAL_STRING = "Copyright"
 HEADER_IN_FIRST_N_LINES = 20
+HEADER_SIGNAL_STRING = "Copyright"
+
+import license_handling
 
```

Comment:
Ideally, however, the file program_commands/check_command `imports userfiles_handling` which then wraps back to `import userfiles_handling`

For example,
```

Traceback (most recent call last):
  File "software_dmv.py", line 8, in <module>
    import program_commands.check_command as check_command
  File "/home/harsha_lv/CIG_codes/cig_tools/Licensing_Program/program_commands/check_command.py", line 4, in <module>
    import userfiles_handling
  File "/home/harsha_lv/CIG_codes/cig_tools/Licensing_Program/userfiles_handling.py", line 10, in <module>
    import license_handling
  File "/home/harsha_lv/CIG_codes/cig_tools/Licensing_Program/license_handling.py", line 9, in <module>
    import config_handling
  File "/home/harsha_lv/CIG_codes/cig_tools/Licensing_Program/config_handling.py", line 28, in <module>
    "maximum": userfiles_handling.HEADER_IN_FIRST_N_LINES,
AttributeError: module 'userfiles_handling' has no attribute 'HEADER_IN_FIRST_N_LINES'
```

The following functions are defined in userfiles_handling and are called in check_command module:

- userfile_handling.sanitize_path
- userfile_handling.userfiles_iter
- userfiles_handling.remove_ignored_userfiles
- userfiles_handling.commentfmt_userfiles_pairing
- userfiles_handling.match_header_in_file